### ISSUE TITLE
Flight control is broken

### ISSUE THREAD

**Issue by MachineMuse** [MachineMuse/MachineMusePowersuits#521]
>pressing space only prevents you from falling, it doesn't raise you (Meaning you go up 1 block when jumping, and then stay at that height) and releasing space makes you just fall. If there's vertical movement active, and space is still pressed, the player does go up while flight control is active

**Comment by MachineMuse** [comment]
Hello,

I found the only way I could reproduce the behaviour was by setting thrust to less than is necessary to be able to maintain flight. Let me know if this keeps happening.

**Comment by sdkee** [comment]
There was an actual problem (not the low thrust setting, I have used this mod before).  Real life issues made me stop playing that world.  The last thing I tried was rebuilding the pack from the most recent forge, and seem to remember that this had solved the basic problem.  I didn't get the chance to import the world into the rebuilt pack to see whether it was forge or the world data causing it.

So I still have no idea what was causing the problem.  Anyway, thanks for the support and sorry I couldn't help isolate further.  My new world is using QMX fork, which is mostly working so far.  Hoping for an official 1.7.10 release soon (and hoping some plan for MPS addons).
